easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCCcore-10.3.0.eb

```
@@ -0,0 +1,59 @@
+easyblock = 'Bundle'
+
+name = 'poppler'
+version = '21.06.1'
+
+homepage = 'https://poppler.freedesktop.org'
+description = "Poppler is a PDF rendering library"
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('freetype', '2.10.4'),
+    ('fontconfig', '2.13.93'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('libpng', '1.6.37'),
+    ('NSS', '3.65'),
+    ('LibTIFF', '4.2.0'),
+    ('Qt5', '5.15.2'),
+    #('Boost', '1.76.0'),
```

This should be fixed, but that requires #13054 ...

---

@boegel seems like we're still stuck on https://github.com/easybuilders/easybuild-easyblocks/pull/2456

---

Boost was merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/12977
```suggestion
    ('Boost', '1.76.0'),
```